src/main/java/core/basesyntax/figure/core/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.figure.core;
+
+public abstract class Figure {
+    private String colorOfFigure;
+
+    public Figure(String colorOfFigure) {
+        this.colorOfFigure = colorOfFigure;
+    }
+
+    public String getColorOfFigure() {
+        return colorOfFigure;
+    }
+
+    public void setColorOfFigure(String colorOfFigure) {
+        this.colorOfFigure = colorOfFigure;
+    }
+
+    public abstract double obtainTheArea();
+
+    public abstract void drawFigure();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes)